project/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.6
+COPY . /app
```

Comment:
Imagine that you want to change `queries.tsv` to be used by service. For that you will need to rebuild your image. It's also best practice to keep your image as light as possible. You usually mount your data as volume when your run the container later with command `docker run`.

Reply:
So I have to exclude `queries.tsv` from my image and mount it separately?

Reply:
Agreed. Fixed with -v